pkg/resource/persistence/schema_setup_job_builder.go

```diff
@@ -95,6 +95,7 @@ func (b *SchemaJobBuilder) Build() (client.Object, error) {
 			Annotations: metadata.GetAnnotations(b.instance.Name, b.instance.Annotations),
 		},
 		Spec: batchv1.JobSpec{
+			TTLSecondsAfterFinished: &b.instance.Spec.JobTtlSecondsAfterFinished,
```

Comment:
```suggestion
			TTLSecondsAfterFinished: b.instance.Spec.JobTtlSecondsAfterFinished,
```

(According to my previous suggestion)